src/controllers/admin.controller.js

```diff
@@ -5,67 +5,71 @@ const universityServiceAPI = require("../services/at-university-api.service");
 
 // AT-UNIVERSITY - Admin - Index
 adminCtrl.renderIndexAdmin = (req, res) => {
-    res.render("admin/index");

```

Comment:
This is not a valid change (blank space or indent code),  is this a new functionality?, if not there is no point in modifying it